etc/scripts/renew_certs_and_redeploy.sh

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env bash
+
+# Set absolute script dir, allows to find absolute project dir & env files
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# Removing "/etc/scripts" suffix gives project dir
+BENTO_PROJECT_DIR="${SCRIPT_DIR%%/etc/scripts}"
+
+# CD to project dir and source env config
+cd $BENTO_PROJECT_DIR && source etc/default_config.env && source local.env;
+
+# Activate python venv
+source env/bin/activate
+
+# Init services array, gateway is always renewed
+SERVICES=('gateway')
+
+# Auth needs cert renewal when internal only
+if [[ $BENTOV2_USE_EXTERNAL_IDP -eq 0 ]]; then
```

Comment:
it might be a bit safer here to check for `!= 1` since we should assume external auth by default, probably